src/app/sections/header/header.component.scss

```diff
@@ -0,0 +1,37 @@
+.headerBar, .menuAction, .menuButton {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+.headerBar {
+  width: 100%;
+  height: 90px;
+  background-color: black;
```

Comment:
Let's add a color variables file like [this](https://sass-lang.com/documentation/variables).